src/System.Memory/src/System.Memory.csproj

```diff
@@ -19,7 +19,10 @@
     <Compile Include="System\Buffers\ReadOnlySequence.cs" />
     <Compile Include="System\Buffers\ReadOnlySequenceDebugView.cs" />
     <Compile Include="System\Buffers\ReadOnlySequenceSegment.cs" />
-    <Compile Include="System\Buffers\ReadOnlySequence_helpers.cs" />
+    <Compile Include="System\Buffers\ReadOnlySequence.helpers.cs" />
```

Comment:
The prevalent style in CoreFX is to start the suffix for partial classes with upper case, i.e. `ReadOnlySequence.Helpers.cs`

Reply:
Changed